src/main/resources/locales/cy/translation/contact-the-tribunal.json

```diff
@@ -74,8 +74,9 @@
       "link": "Welsh Translation required"
     },
     "documents": {
-      "label": "Welsh Translation required",
-      "body": "Welsh Translation required"
+      "label": "Submit documents for a hearing",
```

Comment:
Are these intended or should we put "Welsh Translation required" like before?

Reply:
Fixed